R/plot_with_settings.R

```diff
@@ -607,7 +607,7 @@ type_download_srv <- function(id, plot_reactive, plot_type, plot_w, default_w, p
 #' @return A dataframe of selected rows.
 #' @export
 #'
-clean_brushedPoints <- function(data, brush) { # nolintr
+clean_brushedPoints <- function(data, brush) { # nolint object_name_linter
```

Comment:
According to linter documents it needs a full stop in the end.

Otherwise, it's interpreted as `# nolint`

https://lintr.r-lib.org/reference/exclude.html#details

```suggestion
clean_brushedPoints <- function(data, brush) { # nolint object_name_linter.
```

![image](https://github.com/insightsengineering/teal.widgets/assets/211358/c5ad3cbf-4bf6-4528-83e4-4abda1e8f4db)

Reply:
thank you! TIL this!

Reply:
whoops - misclick :( 
I will open a new one shortly